src/symbolize/symbolizer.rs

```diff
@@ -166,6 +193,19 @@ impl Builder {
         self
     }
 
+    /// Enable/disable source code location information (line numbers,
+    /// file names etc.).
+    pub fn enable_src_location(mut self, enable: bool) -> Builder {
+        self.src_location = enable;
+        self
+    }
+
+    /// Enable/disable inlined function reporting.
+    pub fn enable_inlined_fns(mut self, enable: bool) -> Builder {
```

Comment:
it's a bit of eye sore to read "inline_fns", would "enable_inlines" be confusing? Then you can also call the field in Sym "inlines" and be consistent in naming throughout?

Reply:
> Then you can also call the field in Sym "inlines"

It's actually called `inlined`. And it currently is of type `InlinedFn`. I am not sure I want to shorten it to just a single word, to be honest. Similar for `code info` or `debug syms`. So if anything I'd rather change `Sym::inlined` to `Sym::inlined_fns`. I have no strong preference. The types involved basically add the necessary context from my perspective.